spec/system/school/courses/target_content_editor_spec.rb

```diff
@@ -377,6 +377,73 @@ def file_path(filename)
 
       expect(page).to have_text('https://vimeo.com/123456789')
     end
+
+    context 'when video uploaded without a title' do
+      before do
+        stub_request(:post, 'https://api.vimeo.com/me/videos/')
+          .with(
+            body:
+              "{\"upload\":{\"approach\":\"tus\",\"size\":588563},\"privacy\":{\"embed\":\"whitelist\",\"view\":\"#{account_type == 'basic' ? 'anybody' : 'disable'}\"},\"embed\":{\"buttons\":{\"like\":false,\"watchlater\":false,\"share\":false},\"logos\":{\"vimeo\":false},\"title\":{\"name\":\"show\",\"owner\":\"hide\",\"portrait\":\"hide\"}},\"name\":\"#{target.title}\",\"description\":\"#{description}\"}",
+            headers: request_headers
+          )
+          .to_return(status: 200, body: request_body.to_json, headers: {})
+      end
+
+        scenario 'school admin uploads a video without a title' do
```

Comment:
I think this `scenario` can be removed. This is already tested above and need not be repeated here, as we already test the ability of the school admin to do all these basic operations related to Vimeo uploads. In this particular context, we should just be interested in checking whether the title is set as per the current change.

The course author attempting to do an upload without a title should just suffice